roles/internal/grok/files/etc/ld.so.conf.d/libgrok.conf

```diff
@@ -0,0 +1 @@
+/usr/local/lib
```

Comment:
Is this actually needed? I believe `/usr/local/lib` is in the library path by default.

Reply:
I seemed to need it to decode JP2s, but Tiffs worked without it. Perhaps I just need the to run `ldconfig`?

Reply:
For sure will need `ldconfig` to load the libraries.

I think it should be included by default, I guess its not a big deal to define it twice.